db/migrate/20240909215413_devise_create_users.rb

```diff
@@ -3,17 +3,27 @@
 class DeviseCreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
+
```

Comment:
Não se deve fazer alteração em migrações que já foram mergeadas á main
Deve-se criar uma nova migração alterando o que for necessário sem mexer na antiga